posthog/models/dashboard.py

```diff
@@ -83,6 +83,15 @@ class PrivilegeLevel(models.IntegerChoices):
     def __str__(self):
         return self.name or str(self.id)
 
+    class Meta:
+        indexes = [
+            models.Index(
+                name="idx_dashboard_deleted_team_id",
+                fields=["-pinned", "name", "deleted", "team_id"],
+                condition=models.Q(deleted=False),
```

Comment:
so, we exclude deleted and then sort by them, or sort by them and then exclude them?

worried that the condition means the index won't apply (you might already know that it's fine in which case ignore me :))

Reply:
This partial index now only includes non-deleted, so Postgres can cleverly use it when we have `where deleted=false`